src/main/java/net/snowflake/client/config/SFClientConfigParser.java

```diff
@@ -110,6 +110,8 @@ static String convertToWindowsPath(String filePath) {
       filePath = filePath.substring(1);
     } else if (filePath.startsWith("file:\\")) {
       filePath = filePath.substring(6);
+    } else if (filePath.startsWith("nested:\\")) {
+      filePath = filePath.substring(8);
```

Comment:
Not critical, but maybe we can have `filePath.substring("nested:\\".length)`?

Reply:
I think we can leave it consistent with previous code